Source/SourceKittenFramework/SourceKitVariant.swift

```diff
@@ -0,0 +1,674 @@
+        static let docDiscussion: UID = "key.doc.discussion"
+        static let docFile: UID = "key.doc.file"
+        static let docLine: UID = "key.doc.line"
+        static let docName: UID = "key.doc.name"
+        static let docParameters: UID = "key.doc.parameters"
+        static let docResultDiscussion: UID = "key.doc.result_discussion"
+        static let docType: UID = "key.doc.type"
+        // static let usr: UID.Key = "key.usr"
+        static let parsedDeclaration: UID = "key.parsed_declaration"
+        static let parsedScopeEnd: UID = "key.parsed_scope.end"
+        static let parsedScopeStart: UID = "key.parsed_scope.start"
+        static let swiftDeclaration: UID = "key.swift_declaration"
+        static let alwaysDeprecated: UID = "key.always_deprecated"
+        static let alwaysUnavailable: UID = "key.always_unavailable"
+        static let deprecationMessage: UID = "key.deprecation_message"
+        static let unavailableMessage: UID = "key.unavailable_message"
+    }
+
+    /// Column where the token's declaration begins (Int).
+    public var docColumn: Int? {
+        get { return self[Custom.docColumn]?.int }
+        set { self[Custom.docColumn] = SourceKitVariant(newValue) }
+    }
+
+    /// Documentation comment (String).
+    public var documentationComment: String? {
+        get { return self[Custom.documentationComment]?.string }
+        set { self[Custom.documentationComment] = SourceKitVariant(newValue) }
+    }
+
+    /// Declaration of documented token (String).
+    public var docDeclaration: String? {
+        get { return self[Custom.docDeclaration]?.string }
+        set { self[Custom.docDeclaration] = SourceKitVariant(newValue) }
+    }
+
+    /// Discussion documentation of documented token ([SourceKitVariant]).
+    public var docDiscussion: [SourceKitVariant]? {
+        get { return self[Custom.docDiscussion]?.array }
+        set { self[Custom.docDiscussion] = SourceKitVariant(newValue) }
+    }
+
+    /// File where the documented token is located (String).
+    public var docFile: String? {
+        get { return self[Custom.docFile]?.string }
+        set { self[Custom.docFile] = SourceKitVariant(newValue) }
+    }
+
+    /// Line where the token's declaration begins (Int).
+    public var docLine: Int? {
+        get { return self[Custom.docLine]?.int }
+        set { self[Custom.docLine] = SourceKitVariant(newValue) }
+    }
+
+    /// Name of documented token (String).
+    public var docName: String? {
+        get { return self[Custom.docName]?.string }
+        set { self[Custom.docName] = SourceKitVariant(newValue) }
+    }
+
+    /// Parameters of documented token ([SourceKitVariant]).
+    public var docParameters: [SourceKitVariant]? {
+        get { return self[Custom.docParameters]?.array }
+        set { self[Custom.docParameters] = SourceKitVariant(newValue) }
+    }
+
+    /// Doc result discussion ([SourceKitVariant]).
```

Comment:
These keys comes from SwiftDocKey.swift.
I noticed that documentation comments of them are incorrect.
I copied and modified them to here, but originals are not modified.
https://github.com/jpsim/SourceKitten/blob/master/Source/SourceKittenFramework/SwiftDocKey.swift#L50-L87